packages/obonode/obojobo-chunks-materia/editor-component.js

```diff
@@ -68,13 +74,13 @@ class MateriaEditor extends React.Component {
 	// set text held in the slate text node
 	// used wen returning from the settings dialog to set the title/caption
 	// using slate text so you can click on the caption to edit it
-	setChildText(text) {
-		const path = ReactEditor.findPath(this.props.editor, this.props.element.children[0])
-		// clear out text contents
-		Transforms.delete(this.props.editor, { at: path, unit: 'line' })
-		// set text into slate children text
-		Transforms.insertText(this.props.editor, text, { at: path })
-	}
+	// setChildText(text) {
```

Comment:
There is an issue with this, and it's hard to debug if you don't have Materia running on your system to test. But when you go to select a widget it will not only make the title of that widget the caption text in the dialog, but also makes that title the caption when you close the dialog. This change still shows the caption text as the widget title in the dialog, but when you close the dialog the actual slate caption underneath the node is blank.

Reply:
Got it. That makes sense. Should I leave this issue for those who have Materia running locally then or is it possible to keep working on it without Materia?